src/views/EditPerson/TeamAndJob/index.jsx

```diff
@@ -0,0 +1,167 @@
+import React, { useState, useEffect } from "react";
+import { useSelector } from "react-redux";
+import _ from "lodash";
+import TreeList from "../../../components/groups/groupTree";
+import {
+  TextField,
+  InputAdornment,
+  Link,
+  ExpansionPanel,
+  ExpansionPanelSummary,
+  ExpansionPanelDetails,
+  Typography,
+  Divider,
+  ClickAwayListener,
+} from "@material-ui/core";
+import { ExpandMore } from "@material-ui/icons";
+import styles from "./TeamAndJob.style";
+import PropTypes from "prop-types";
+
+export default function TeamAndJob({ formInputs, onChangeHandle, personDetails, disabled }) {
+  // jss style
+  const classes = styles();
+  // RootId of tree from redux
+  const rootIds = useSelector((state) => state.groups.rootGroupsIds);
+  // Array of expended in tree
+  const [treeExpanded, setTreeExpanded] = useState(rootIds);
+  // Selected group 
+  const [treeSelected, setTreeSelected] = useState(null);
+  // expended expention toggle
+  const [expentionExtended, setExpentionExtended] = useState(false);
+  // group of person from redux
+  const groupById = useSelector((state) => state.groups.byId[treeSelected]);
+  const [hierarchyDisplay, setHierarchyDisplay] = useState("בחר צוות");
+  const handleExpandedChange = (e, nodes) => setTreeExpanded(nodes);
+  // Wether select group is change in input and tree 
+  const handleSelection = (e, node) => {
+    const event = { target: { name: "directGroup", value: node } };
+    onChangeHandle(event);
+    setTreeSelected(node);
+  };
+  // When are rootids loadded in redux, the tree expend them
+  useEffect(() => {
+    setTreeExpanded(rootIds);
+  }, [rootIds]);
+  // when are personalDetails changed, the expention closed
+  useEffect(() => {    
+    setTreeSelected(null);
+  }, [personDetails]);
+  // hierarchy display changed acording to group of person and selected 
+  useEffect(() => {
+    let hierarchyDisplay = _.isEmpty(personDetails)
+      ? "בחר צוות"
+      : groupById
+      ? groupById.hierarchy.concat(groupById.name).join(" / ")
+      : !_.isEmpty(personDetails)
+      ? personDetails.hierarchy.join(" / ")
+      : null
+    setHierarchyDisplay(hierarchyDisplay);
+  }, [groupById, personDetails]);
+
+  // Indicate if disabled input
+  let disabledInput = disabled || _.isEmpty(personDetails);
+  return (
+    <div className={classes.teamAndJobContainer}>
+      <div className={classes.expansionPanelContainer}>
+        <span>
+          <strong>צוות ותפקיד</strong> השלם את הפרטים הבאים:
```

Comment:
why STRONG

Reply:
Why not?

Reply:
From https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong: 
"Typically this element is rendered by default using a bold font weight. However, it should not be used simply to apply bold styling; use the CSS font-weight property for that purpose"